test/scripts/window.py

```diff
@@ -1,105 +1,52 @@
 #!/usr/bin/env python3
-# Copyright (c) 2011 Florian Mounier
-# Copyright (c) 2014 Sean Vig
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-# SOFTWARE.
 """
-    This program is carefully crafted to exercise a number of corner-cases in
-    Qtile.
+This creates a minimal window using GTK that works the same in both X11 or Wayland.
 
-    Usage:
-        window.py DISPLAY WINDOW_NAME [WINDOW_TYPE]
+GTK sets the window class via `--name <class>`, and then we manually set the window
+title and type. Therefore this is intended to be called as:
+
+    python window.py --name <class> <title> <type>
+
+where <type> is "normal" or "notification"
+
+The window will close itself if it receives any key or button press events.
 """
-import sys
-import time
+# flake8: noqa
```

Comment:
Why tho? :D

Reply:
The imports need to be in a non-standard order, and there doesn't appear to be a way to ignore the error more specifically.